reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+
+Most community groups do not have a voting process as they are driven by 
+consensus from OWNERs files and the community-at-large. However, in some cases, 
+particularly leadership transitions, community groups voted to decide.
+
+**Sustainable Membership**
+
+While onboarding new contributors is a critical part of a sustainable open
+source project, contributors shift focus, change jobs, or
+step away from the project for a variety of other reasons. In late 2019, the 
+Steering Committee updated our guidelines to include an [emeritus status]. 
+Emeritus community members have stepped away from the project, but are still 
+recognized for their work. Giving people the ability to step away gracefully 
+helps ensure an overall healthy and active community.
+
+In 2020, 185 members stepped down or were removed for inactivity (18
+months with 0 activity across any GitHub organization). The Kubernetes project 
+has around 1300 active members at any given time.   
+
+Joining the project's GitHub organizations is celebrated, but
+there are no regular means of celebration or recognition for those that join a
+SIG, community group, or [achievement for climbing the ladder].
+
+[elections]: https://github.com/kubernetes/steering/blob/master/elections.md#eligibility-for-voting
+[membership]: https://git.k8s.io/community/community-membership.md#member
+[emeritus status]: https://k8s.dev/docs/guide/owners/#emeritus
+[achievement for climbing the ladder]: https://github.com/kubernetes/test-infra/issues/11994
+[charter process]: https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-charter-template.md
+
+
+#### Areas of research
+
+- Automation around suggesting when contributors may be ready to move to the
+  next step in the lifecycle
+- Ways to promote a sustainable flow through the various stages of the lifecycle
+- Clarifying and/or simplifying definitions around the stages of the lifecycle
+- Prompts to assist contributors in knowing what they need to do to 'level up'
+  to the next stage in the lifecycle (e.g. prompts when you join a mailing list)
+
+
+### Growing a diverse group of OWNERs
+
+Growing on the contributor ladder to ownership doesn't happen immediately after
+your first contribution. Building trust is a key part of our values and is a
+step in the "contributor journey", called the Kubernetes [contributor
+ladder][ldr]. We need more diverse, trusted folks from all backgrounds to grow
+in these areas. Balancing being welcoming and identifying contributors to
+encourage to stick around at scale is difficult, especially with tens of
+thousands of casual contributors (yes, a small city) and many cultures that come
+together. 30 new leaders have stepped up to Chair and other roles in 2020, but
+bandwidth for the reviewers and approvers in many subprojects remains a
+challenge as does the diversity of those folks.
+
+The Groups utilize the following to find and grow contributors:
```

Comment:
```suggestion
#### What we've done
```

Can we label this "What we've done"? See notes on making the formatting of all these sub-sections under Themes the same.